### ISSUE TITLE
Price not formatted in cart item instance

### ISSUE THREAD

**Issue by bloggervista** [darryldecode/laravelshoppingcart#114]
$item->price so raw price not with comma spaces etc.. Why?

Thanks for this awesome package.

**Comment by frezno** [comment]
take a look here, @bloggervista 
https://github.com/darryldecode/laravelshoppingcart#format-response

it might be what you are looking for. not sure though, since i don't use this but wrote my own helper for formatting the price.

**Comment by bloggervista** [comment]
@frezno  That is done by config . I mean $item->price etc are not formatted but Cart::total() is formatted. I guess we need to improve that Cart::getContent() instance.

**Comment by frezno** [comment]
got it - as said, i use my own helper, so i wasn't aware about this.

**Comment by bloggervista** [comment]
yeah i guess if it is applied to Cart::total() it should have been applied for item instance to make api same for item price or for total price. I thought it was my mistake as it was not givin same result. Yes now i also created my own helper function :)

**Comment by malaymehta** [comment]
@darryldecode This also can be closed.

**Comment by ezani92** [comment]
bump!
